### ISSUE TITLE
Error Including 'augmentSchema' : Duplicate "graphql" modules

### ISSUE THREAD

**Issue by JobinMathewP** [neo4j-graphql/neo4j-graphql-js#81]
```
Error: Cannot use GraphQLObjectType "__Directive" from another module or realm.

Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.

https://yarnpkg.com/en/docs/selective-version-resolutions

Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
    at instanceOf (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/node_modules/graphql/jsutils/instanceOf.js:37:13)
    at isObjectType (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/node_modules/graphql/type/definition.js:93:34)
    at isNamedType (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/node_modules/graphql/type/definition.js:338:32)
    at isSpecifiedScalarType (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/node_modules/graphql/type/scalars.js:156:38)
    at isDefinedType (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/node_modules/graphql/utilities/schemaPrinter.js:57:46)
    at Array.filter (<anonymous>)
    at printFilteredSchema (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/node_modules/graphql/utilities/schemaPrinter.js:65:6)
    at printSchema (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/node_modules/graphql/utilities/schemaPrinter.js:47:10)
    at addMutationsToSchema (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/dist/augmentSchema.js:37:52)
    at augmentSchema (/home/mjobin0/dev/NeoGraphQL/node_modules/neo4j-graphql-js/dist/index.js:358:52)
```

**Comment by JobinMathewP** [comment]
```
const { neo4jgraphql, augmentSchema } = require('neo4j-graphql-js')

const augmentedSchema = augmentSchema(schema);
```

**Comment by johnymontana** [comment]
This should have been fixed by b8a0a7f6a7698ac4c83690ed8950bca892d11c93

Please reopen if you experience this error again.
